lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -97,6 +97,21 @@ defmodule TrentoWeb.V1.SapSystemView do
 
   def render("application_instance_health_changed.json", %{health: health}), do: health
 
+  def render("instance_absent_at_changed.json", %{
```

Comment:
I would follow the same pattern we use in others and wrap the data in a `instance` field, like what we do in `database_instance_health_changed.json` and `database_instance_system_replication_changed.json"`